test/decorators/context.test.ts

```diff
@@ -0,0 +1,40 @@
+import { Context as AzureContext, HttpRequest } from '@azure/functions';
+import { Context, HttpFunction, PathParameter } from '../../src';
+
+type ContextEchoResponse = {
+    page: number;
+    context: AzureContext;
+};
+
+class ContextEcho {
+    @HttpFunction()
+    static async httpTrigger(
+        @PathParameter('page') page: number,
+        @Context() context: AzureContext
+    ): Promise<ContextEchoResponse> {
```

Comment:
It is now possible to specifically pass the Context argument, always provided as the first argument by azure, to a special positined argument.